library/spdm_responder_lib/libspdm_rsp_algorithms.c

```diff
@@ -188,6 +188,12 @@ return_status spdm_get_response_algorithms(IN void *context,
 
 	ext_alg_total_count = 0;
 
+	ASSERT(!((((spdm_context->local_context.capability.flags >> 3) & 0x03) == 0) ^
```

Comment:
Please use SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_MEAS_CAP.

Reply:
Yes. I have changed the code to use SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_MEAS_CAP. Thanks.